src/strings/PowerFxResources.en-US.resx

```diff
@@ -5453,4 +5453,8 @@
     <value>Table doesn't contain any column of text type.</value>
     <comment>Error Message.</comment>
   </data>
+  <data name="ErrDeprecatedDotUseShowColumn" xml:space="preserve">
```

Comment:
>ShowColumn [](http://example.com/codeflow?start=33&length=10)

[nit] the function name is ShowColumns (with an 'S' at the end), we should be consistent.